src/main/java/com/TableTalk/Enterprise/dao/IGameDAO.java

```diff
@@ -7,9 +7,35 @@
 import java.util.List;
 
 public interface IGameDAO {
+    /**
+     * Generates a game
+     *
+     * @param game
+     * @return
+     * @throws Exception
```

Comment:
I could figure all of this out from reading the method, so it's not adding much value.  How I'd improve it:

- What does the method do?
- What are the parameters?
- What are the return values?
- Under what circumstances would an exception be thrown?